common/lib/dependabot/config/ignore_condition.rb

```diff
@@ -76,11 +76,9 @@ def ignore_major(version)
 
         version_parts = version.split(".")
         lower_parts = [version_parts[0].to_i + 1] + ["a"]
-        upper_parts = [version_parts[0].to_i + 2]
         lower_bound = ">= #{lower_parts.join('.')}"
-        upper_bound = "< #{upper_parts.join('.')}"
 
-        ["#{lower_bound}, #{upper_bound}"]
+        [lower_bound.to_s]
```

Comment:
```suggestion
        [lower_bound]
```

Nit: the `.to_s` seems superflous, we initialized the string on L79 so it's probably still a string.

Noice: I dig keeping the `lower_bound` naming+structure for consistency.

Reply:
Nice catch! That was an auto rubocop fix from `["#{lower_bound}"]` so I didn't think about it.